src/reducers/index.js

```diff
@@ -209,6 +255,51 @@ function initializeRowHeightsAndOffsets(state) {
   });
 }
 
+function initializeColWidthsAndOffsets(state) {
+  const { columnProps } = state;
+  let { columnSettings } = state;
+  const columnsCount = columnProps.length;
+  let scrollContentWidth = 0;
+  let fixedColumnsWidth = 0;
+  let fixedRightColumnsWidth = 0;
+  const storedWidths = [];
+  let minColumnWidth = -1;
+
+  for (let idx = 0; idx < state.scrollableColumns.cell.length; idx++) {
```

Comment:
Right now, this iterates over all scrollable columns from props even though the purpose of virtualization is to only do it for the columns in the viewport.
This is temporary, once https://github.com/karry08/fixed-data-table-2/pull/2/ is ported, we'll only do this for the visible columns in the viewport.